core/trino-server-rpm/pom.xml

```diff
@@ -11,11 +11,16 @@
 
     <artifactId>trino-server-rpm</artifactId>
     <packaging>rpm</packaging>
+    <description>Trino Server RPM package</description>
+
+    <!-- will be used by the rpm-builder -->
+    <url>https://trino.io</url>
```

Comment:
Shouldn't this be inherited from the parent POM?

Reply:
It seems that it's not

Reply:
That's actually interesting. If I ran on master:

```
./mvnw help:effective-pom -pl ':trino-server' | grep -i url
Running `/Users/mateuszgajewski/Projects/src/github.com/trinodb/trino/mvnw`...
  <url>https://trino.io/core/trino-server</url>
```

Reply:
So it's preexisting condition

Reply:
According to the docs from maven xsd:

```
The URL to the project's homepage.
            <br><b>Default value is</b>: parent value [+ path adjustment] + (artifactId or project.directory property), or just parent value if
            project's <code>child.project.url.inherit.append.path="false"</code>
```

Reply:
https://github.com/trinodb/trino/pull/20750